### PR TITLE
Fix click v7 version_option compatibility

### PR DESCRIPTION
**Pull request by FuegoFro**
It looks like the ternary was committed backwards in https://github.com/jazzband/pip-tools/pull/1400, since the `package_name` option was add in click 8, but is only being passed in click 7. That caused `pip-tools` to error on import when installed alongside click 7.

**Changelog-friendly one-liner**: Follow up https://github.com/jazzband/pip-tools/pull/1400 to fix click v7 compatibility.

##### Contributor checklist

- [ ] Provided the tests for the changes.
- [x] Gave a clear one-line description in the PR (that the maintainers can add to CHANGELOG.md on release).
- [x] Assign the PR to an existing or new milestone for the target version (following [Semantic Versioning](https://blog.versioneye.com/2014/01/16/semantic-versioning/)).

### CODE REVIEW

**Comment by FuegoFro** [comment]
Hey there! I wasn't sure how to set a milestone or what milestone should even be set (possibly 6.1.1, just since that's the milestone which was set on https://github.com/jazzband/pip-tools/pull/1400?). Would love some help with that!

Also, I wasn't sure whether I should add a new `tox` env to tests this, let me know if you think that'd be valuable!

**Review by nicoa — APPROVED** [review]

**Comment by nicoa** [comment]
you're totally right, good catch! As far as I know, only members of the jazzband organisation can set milestones, I added 6.1.1 for now, agreeing that this makes sense.

Regarding the env: I honestly don't know, maybe others have a strong opinion on this.

**Review by webknjaz — APPROVED** [review]

**Comment by webknjaz** [comment]
This will probably need to be synced with master to unblock the merge button.

**Comment by FuegoFro** [comment]
Thank you for the quick review! 😀 I've gone ahead and rebased the branch on latest `master` 🙂

**Pull request merged**
